packages/cli/schema.prisma

```diff
@@ -0,0 +1,14 @@
+datasource db {
```

Comment:
This schema file at the root of the CLI package is unexpected to me.
Could you remove it? (Is it needed for anything?)